classes/user/report/QueryBuilder.inc.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @defgroup lib_pkp_classes_user
+ */
+
+/**
+ * @file lib/pkp/classes/user/report/QueryBuilder.inc.php
+ *
+ * Copyright (c) 2003-2020 Simon Fraser University
+ * Copyright (c) 2003-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file LICENSE.
+ *
+ * @class QueryBuilder
+ * @ingroup lib_pkp_classes_user
+ *
+ * @brief Builds the query to retrieve the users.
+ */
+
+namespace PKP\User\Report;
+use \PKP\Services\QueryBuilders\PKPUserQueryBuilder;
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class QueryBuilder extends PKPUserQueryBuilder {
```

Comment:
I'd like to move most of the code here to the `PKPUserQueryBuilder`, but instead of many left joins, I'd like to use what I've proposed in the beginning of this discussion.